nservicebus/hosting/cloud-services-host/shared-hosting.md

```diff
@@ -13,7 +13,7 @@ redirects:
  - nservicebus/azure/shared-hosting-nservicebus-in-windows-azure-cloud-services
 ---
 
-If real scale is needed, as in tens, hundreds or even thousands of machines hosting each endpoint, than cloud services is the suitable deployment model. But very often, one only wants this scale when the project is eventually successful, not when just starting out. To support this scenario, we've created the `AsA_Hos`t endpoint role for Azure cloud services.
+If real scale is needed, as in tens, hundreds or even thousands of machines hosting each endpoint, than cloud services is the suitable deployment model. But very often, one only wants this scale when the project is eventually successful, not when just starting out. To support this scenario, we've created the `Host` endpoint role for Azure cloud services.
```

Comment:
one only wants this scale when the project is eventually successfull, not when just starting out.